src/test/resources/features/comp/COMPEndToEnd.feature

```diff
@@ -0,0 +1,41 @@
+@COMPEndToEnd @COMP
+Feature: COMP End To End
```

Comment:
Why do we need to have all the scenarios as -> Scenario: User can complete and close a COMP case 
tests the end to end. By having all the scenarios it increases the regression time and test the same flow again and again.

Reply:
These scenarios dont have the regression tag. We make these just to be helpful things in the pack if we need to manually trigger a test to go through a stage, they arent intended for regressions

Reply:
In that case can we have @COMPRegression tag to the Scenario: User can complete and close a COMP case